partials/footer-content.php

```diff
@@ -54,7 +54,15 @@
                 <ul>
                     <li><a href="<?php echo $site_url ?>/legal/">Terms of use</a></li>
                     <li><a href="<?php echo $site_url ?>/legal/privacy.htm">Privacy policy</a></li>
-                    <li><a href="<?php echo $site_url ?>/legal/cookies.htm">Cookies</a></li>
+                    <?php
+                        $siteUrl = site_url();
+
+                        if (strpos($siteUrl, 'latin') !== false) { ?>
```

Comment:
It took me a while to see that the difference between these two links was 'latin' vs 'legal'. Would it be possible to assign the URL to a variable and have a single link instead? I'm just thinking that it might be clearer for people to see what's going on in future?